provider-ui/cypress.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       return require('./cypress/plugins/index.js')(on, config)
     },
-    baseUrl: 'http://localhost:3001',
+    baseUrl: 'http://localhost:3001/provider',
```

Comment:
Let's consider a separate improvement (separate issue) to dynamically attempt to use a higher port number in the case of port conflict on the contributor's system.

Reply:
I reverted the change, it was not intended to be this way.